ExampleLPG2RDF.ttl

```diff
@@ -30,6 +30,12 @@ PREFIX ex:     <http://example.org/>
 #ex:IRIBasedEdgeLabelMapping a rdfs:Class ;
 #   rdfs:subClassOf ex:EdgeLabelMapping .
 
+#ex:PropertyMapping a rdfs:Class ;
+#
+#ex:IRIBasedPropertyMapping a rdfs:Class ;
+#   rdfs:subClassOf ex:PropertyMapping .
```

Comment:
Can you please call these classes `PropertyNameMapping` and `IRIBasedPropertyNameMapping` in order to be aligned with the terminology used in the formalization of LPG-to-RDF configurations.

Similarly, the property should be renamed from `propertyMapping` to `propertyNameMapping`.

And, the naming in the Java code should be adapted accordingly (e.g., the name of the interface, of the classes, of the member variables).